src/cloud-api-adaptor/test/e2e/common.go

```diff
@@ -139,6 +140,10 @@ func NewBusyboxPod(namespace string) *corev1.Pod {
 	return NewBusyboxPodWithName(namespace, "busybox")
 }
 
+func NewAlpineCurlPod(namespace string) *corev1.Pod {
```

Comment:
As commented in another place, we seems don't need introduce the new alpine pod but can reuse the busybox pod.